_posts/en/newsletters/2021-05-26-newsletter.md

```diff
@@ -106,7 +106,46 @@ answers posted since our last update.*
 {% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME:bitschmidty
+- [Why are there more than two transaction outputs in a coinbase transaction?]({{bse}}105831)
+  Andrew Chow explains some common outputs in a coinbase transaction:
+
+    * a single miner block reward payment
+
+    * multiple payments, as with a mining pool paying miners
+
+    * [BIP141][bip141 commitment]'s `OP_RETURN` witness commitment
+
+    * additional `OP_RETURN` commitments, as in [merge mining][se 273 merge
+      mining] and other protocols
+
+- [fundrawtransaction - what is it?]({{bse}}105811)
+  Pieter Wuille illustrates what the `fundrawtransaction` RPC does by providing
+  4 examples of how to send coins using the RPC.
+
+- [What previously existing technologies made Bitcoin possible?]({{bse}}106000)
+  Murch provides a summary, based on the [Bitcoin's Academic Pedigree
+  paper][bitcoins academic pedigree paper], of the existing technological ingredients
+  that were combined to create Bitcoin. These technologies are linked
+  timestamping/verifiable logs, byzantine fault tolerance, proof of work,
+  digital cash, and public keys as identities.
+
+- [How can I follow the progress of miner signaling for Taproot activation?]({{bse}}105853)
+  In addition to Hampus Sjöberg's [https://taproot.watch][taproot watch website] website, Bitcoin
+  Core users can use `getblockchaininfo` to get a count of signaling blocks and
+  `getblock`'s versionhex field, where the signaling version bits reside, to
+  observe signaling.
+
+- [What block space savings would we get for coinjoins (and payjoins) if we had cross input signature aggregation?]({{bse}}106224)
+  Murch calculates potential WU for P2TR coinjoin transactions for participants
+  utilizing two input, two output payments using cross input signature aggregation as:
+
+  - 1 participant coinjoin per participant cost: 783 WU
```

Comment:
Maybe call this "Base cost for two inputs and two outputs" since a "1 participant coinjoin" doesn't make sense by itself.

Reply:
Also, our style guide says "use vbytes rather than weight units, unless using weight units would significantly improve conciseness or comprehensibility."  You can just divide all of @Xekyo's figures by 4 to get vbytes (up to you whether or not to include any decimal remainders; I usually include them).

If you really want to use weight units (which is fine by me), I suggest at least slWU/Weight Units (WU)/ for the first occurrence of the term.

Reply:
This question was removed. Hopefully we can include it in next month's SE section.